### PR TITLE
KCI-136 > build and lint (and package.json lint)

### PR DESCRIPTION
**Pull request by jvhellemondt**
i can't test it :sob: -- or can i?

### CODE REVIEW

**Comment by ricardovanlaarhoven** [comment]
Yes you can!

run `vue create test-scaffold`
and `vue add vuetify`

and then `npm i path\to\scaffold`
followed by `vue invoke kingscode-scaffold`

File: README.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ later in the `vue add vuetify` will be overwritten by vuetify, or the kings code
 ### Vue:
 We'd recommend you to choose
 - babel
+- eslint
+  > eslint-config-airbnb
```
Comment (ricardovanlaarhoven) [inline README.md]: do we? `eslint-config-airbnb`

File: generator/templates/Deployment/_github/workflows/build_and_lint.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    steps:
+      - uses: actions/checkout@main
+
+      - name: build .npmrc
+        env:
+          FONT_AWESOME_TOKEN: ${{ secrets.font_awesome_token  }}
+        run: |
+          echo "@fortawesome:registry=https://npm.fontawesome.com/" >> .npmrc
+          echo "//npm.fontawesome.com/:_authToken=$FONT_AWESOME_TOKEN" >> .npmrc
```
Comment (ricardovanlaarhoven) [inline generator/templates/Deployment/_github/workflows/build_and_lint.yml]: Here should be an if

File: generator/templates/Deployment/_github/workflows/deploy.demo.yml [hunk 3]
```diff
@@ -3,14 +3,14 @@ name: Deploy [production]
 on:
   push:
     branches:
-      - 'env/production'
```
Comment (ricardovanlaarhoven) [inline generator/templates/Deployment/_github/workflows/deploy.demo.yml]: don't change that. Production is the default, not all projects have a demo env
(and also the file name)

Comment (ricardovanlaarhoven) [reply generator/templates/Deployment/_github/workflows/deploy.demo.yml]: But al projects have a production :D

**Review by ricardovanlaarhoven — CHANGES_REQUESTED** [review]
beside my comments:
wouldn't it be cool if this was configured using a checkbox in the prompts? (just like fontawesome)

Then we can remove that run lint from the readme as well and can people choose that themselves

**Review by ricardovanlaarhoven — APPROVED** [review]

**Comment by jvhellemondt** [comment]
> Ricardo
>> beside my comments:
>> wouldn't it be cool if this was configured using a checkbox in the prompts? (just like fontawesome)
>> 
>> Then we can remove that run lint from the readme as well and can people choose that themselves

The `run lint` is in the default Vue README, I'd suggest to keep it (the default README) in favor of future updates on it. However, the checkbox is a very good idea.

**Review by ricardovanlaarhoven — APPROVED** [review]

**Pull request merged**